ocs_ci/ocs/ui/views.py

```diff
@@ -8,6 +8,14 @@
 
 osd_sizes = ("512", "2048", "4096")
 
+SUCCESS_TOAST_SELECTORS = [
```

Comment:
locators are separated not the way we store them in views.py, they need to be a part of the dictionary. Please find one category and fetch them in usual way, via BaseUI attributes. This way they will be fetched accordingly to OCP version